src/components/parameters/ParameterBasePath.svelte

```diff
@@ -2,7 +2,7 @@
 
 <script lang="ts">
```

Comment:
In bananation I can't seem to reset an updated path to the mission modal / sim template
<img width="546" alt="image" src="https://user-images.githubusercontent.com/4419607/231846915-33463345-7bb6-4d05-b806-3974aafe2478.png">

Reply:
Actually the path input doesn't seem to respond to changing the template either

Reply:
Nice catch! I left the listener out accidentally...